src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -791,7 +792,7 @@ public void saveCaseEvent(String authorisation, Long caseId, CaseEvent caseEvent
             );
 
             CCDCase ccdCase = extractCase(startEventResponse.getCaseDetails());
-
+            ccdCase.setState(null);
```

Comment:
Please don't set state to null explicitly.

Reply:
In this particular method I found that we are not using the mappers to map from cmc domain model to ccd model  which is why the case state has to be set to null explicitly. Since we do not maintain the state in ccd definition it throws errors if it is not removed i.e set to null .I am open to any viable suggestions or workarounds.

Reply:
We can use `@JsonProperty(access = JsonProperty.Access.WRITE_ONLY)` on the property itself

Reply:
good one. thanks. Done